full-stack-tests/core/src/frontend/hub/ITwinPlatformEnv.ts

```diff
@@ -87,16 +87,16 @@ export class IModelBankFrontend implements TestFrontendHubAccess {
 export interface ITwinPlatformAbstraction {
   readonly hubAccess: TestFrontendHubAccess;
   readonly contextMgr: ContextRegistryClientWrapper;
-  readonly authClient?: FrontendAuthorizationClient;
+  readonly authClient?: AuthorizationClient;
 }
 
 /** A convenient wrapper that includes a default set of clients necessary to configure an iTwin.js application for the iTwin Platform. */
 export class ITwinPlatformCloudEnv implements ITwinPlatformAbstraction {
   public readonly contextMgr = new ContextRegistryClientWrapper(); // this should be the new ContextRegistryWrapper defined in #2045
   public readonly hubAccess = new IModelHubFrontend();
-  public readonly authClient?: FrontendAuthorizationClient; // This should be the new AuthorizationClient method defined in #
+  public readonly authClient?: AuthorizationClient; // This should be the new AuthorizationClient method defined in #

```

Comment:
Can you remove this comment for me? Looks like it's leftover from the frontend hub changes PR from last week.

Reply:
Should be removed now